lua/autorun/client/cl_votestuff_init.lua

```diff
@@ -0,0 +1,77 @@
+if SERVER then return end;
+
+local filterPattern = "&#?[%w]+;"
+
+surface.CreateFont( "Question-font", {
+	font = "Century Gothic",
+	size = ScreenScale(7),
+	scanlines = 0,
+	antialias = true,
+	extended = true,
+})
+
+surface.CreateFont( "Anwser-font", {
+	font = "Century Gothic",
```

Comment:
is this font on people's PC?

Reply:
Yes, it is. It's the default font on every **windows** PCs. For linux or MAC it will use the default fonts for their system.